build-logic/src/main/kotlin/LicenseFileValidation.kt

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2024 Snowflake Computing Inc.
```

Comment:
Shouldn't this be the Apache license here?

Reply:
Yes correct